public/locale/en.json

```
@@ -2572,6 +2574,8 @@
   "search_by_department_team_name": "Search by department/team name",
   "search_by_emergency_contact_phone_number": "Search by Emergency Contact Phone Number",
   "search_by_emergency_phone_number": "Search by Emergency Phone Number",
+  "search_by_encounter_id": "Search by Encounter ID",
```

Biome (1.9.4)

[error] The key search_by_encounter_id was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)